CardScan/Classes/GeneratedSSD.swift

```diff
@@ -0,0 +1,184 @@
+
+//
+// SSDOcr.swift
+//
+// This file was automatically generated and should not be edited.
+//
+
+import CoreML
+
+
+/// Model Prediction Input Type
+@available(macOS 10.13, iOS 11.0, tvOS 11.0, watchOS 4.0, *)
```

Comment:
I know that this file was auto-generated but all of the forced unwraps make me really really nervous. Can we pick those out by hand? We might need to change the high level interface a bit for it but I think it's worth it.